conan/cache/db/cache_database.py

```diff
@@ -23,8 +23,8 @@ def dump(self, output: StringIO):
         self._references.dump(output)
 
     def update_reference(self, old_ref: ConanReference, new_ref: ConanReference = None,
-                         new_path=None, new_remote=None, new_timestamp=None, new_build_id=None):
-        self._references.update(old_ref, new_ref, new_path, new_remote, new_timestamp, new_build_id)
+                         new_path=None, new_timestamp=None, new_build_id=None):
+        self._references.update(old_ref, new_ref, new_path, new_timestamp, new_build_id)
```

Comment:
You are completely removing the remote information. I thought that we would still store (and only store, do nothing with it, just information) that remote. But if it is not being used, then it might be better to just simplify it, remove from the codebase, and if you need it, with revisions it is a ``conan list`` command and you will get where it is from the remotes. Wdyt @lasote?

Reply:
I thought about removing it too in the branch I'm working on. Let's simplify to the max first, we will have years to complicate it.